docker/Dockerfile

```diff
@@ -33,7 +33,7 @@ FROM centos:7 AS prep_open_search_files
 
 ENV PATH /usr/share/opensearch/bin:$PATH
 RUN curl -s https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_linux-x64_bin.tar.gz | tar -C /opt -zxf -
-ENV JAVA_HOME /opt/jdk-11.0.1
+ENV OPENSEARCH_JAVA_HOME /opt/jdk-11.0.1
```

Comment:
OpenSearch 1.3 already supports OPENSEARCH_JAVA_HOME, container builds and starts with no issues